fpl/data/data_converter.py

```diff
@@ -0,0 +1,79 @@
+"""Convert data to CSV."""
+import json
+from pathlib import Path
+
+import pandas as pd
+from tqdm import tqdm
+
+from fpl.utils import paths
+
+
+class DataConverter:
+    """Data converter class."""
+
+    def __init__(self, entity: str, raw_data_path=None, interim_data_path=None):
+        """Set configuration parameters.
+
+        Args:
+            entity (str): Either "elements" or "teams".
+            raw_data_path (Path, optional): Path to dir holding JSON dumps of raw data.
+            interim_data_path (Path, optional): Path to dir for uploading converted data.
+        """
+        self.entity = entity
+        if raw_data_path is None:
+            raw_data_path = paths.get_raw_data_path()
+        self.__raw_data_path = Path(raw_data_path)
+        if interim_data_path is None:
+            interim_data_path = paths.get_interim_data_path()
+        self.__interim_data_path = Path(interim_data_path)
+        self._interim_data_entity_path = Path(
+            self.__interim_data_path, self.__raw_data_path.name + "_" + self.entity
+        ).with_suffix(".csv")
+
+    def _make_interim_folder_if_absent(self):
```

Comment:
This will fail if other "directories" in path does not exist. 
```python
self._interim_data_entity_path.parent.mkdir(exist_ok=True, parents=True)
```
Consider if parent directories should be created.